README.md

```diff
@@ -15,14 +15,30 @@ Windows OS:
 # :warning: ADVISORY :warning:
 > Before running the steps below, follow the steps [here](https://github.com/Duke-MatSci/materialsmine/blob/develop/resfulservice/misc/README.md) to retrieve and deploy required environment variables
 
+> *IMPORTANT NOTICE:* Install nvm. For windows PC click [windows](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows#install-nvm-windows-nodejs-and-npm), for mac PC `brew install nvm`
```

Comment:
I suggest saying just "for Mac" instead of "for Mac PC" since colloquially people still associate PC with Windows